modules/cnf-configure_for_irq_dynamic_load_balancing.adoc

```diff
@@ -0,0 +1,156 @@
+[id="configuring_for_irq_dynamic_load_balancing_{context}"]
+= Configuring a node for IRQ dynamic load balancing
+
+To configure a cluster node to handle IRQ dynamic load balancing, do the following:
+
+. Log in to the {product-title} cluster as a user with cluster-admin privileges.
+. Set the performance profile to use `apiVersion: performance.openshift.io/v2`.
+. Remove the `GloballyDisableIrqLoadBalancing` field or set it to `false`.
+. Set the appropriate isolated and reserved CPUs. The following snippet illustrates a profile that reserves 2 CPUs:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: dynamic-irq-profile
+spec:
+  cpu:
+    isolated: 2-5
+    reserved: 0-1
+----
+. SSH into the pod configured to use exclusive CPUs, and set `irq-load-balancing.crio.io` and `cpu-quota.crio.io` annotations to `disable`. For example:
```

Comment:
I guess need to be _"Create the pod ..."_ instead of _"SSH into the pod ..."_

Reply:
made this change.